grafana-plugin/src/containers/ServiceNowConfigDrawer/ServiceNowTokenSection.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Button, HorizontalGroup, LoadingPlaceholder, VerticalGroup, useStyles2 } from '@grafana/ui';
+import { observer } from 'mobx-react';
+
+import { IntegrationInputField } from 'components/IntegrationInputField/IntegrationInputField';
+import { RenderConditionally } from 'components/RenderConditionally/RenderConditionally';
+import { Text } from 'components/Text/Text';
+import { ActionKey } from 'models/loader/action-keys';
+import { useCurrentIntegration } from 'pages/integration/OutgoingTab/OutgoingTab.hooks';
+import { useStore } from 'state/useStore';
+import { useIsLoading } from 'utils/hooks';
+
+import { getCommonServiceNowConfigStyles } from './ServiceNow.styles';
+
+
+export const ServiceNowTokenSection: React.FC = observer(() => {
+  const styles = useStyles2(getStyles);
+  const { id } = useCurrentIntegration();
+  const { alertReceiveChannelStore } = useStore();
+  const [isExistingToken, setIsExistingToken] = useState(undefined);
+  const [currentToken, setCurrentToken] = useState(undefined);
```

Comment:
If we do it this way (`useState(undefined)`), then the state gets `any` type which we want to avoid and at some point ditch from the whole codebase.

So we should either give it some other default value like boolean / string or explicitly provide a type like `useState<string | undefined>(undefined)